src/opnsense/mvc/app/views/OPNsense/IPsec/key_pairs.volt

```diff
@@ -0,0 +1,123 @@
+{#
+Copyright (C) 2019 Pascal Mathis <mail@pascalmathis.com>
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice,
+   this list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+#}
+
+<script>
+  $(document).ready(function () {
+    const $applyLegacyConfig = $('#applyLegacyConfig');
```

Comment:
be careful with that in `core`. I prefer ES6 too but I don't know how long @AdSchellevis and @fichtner still want to support outdated browsers here. They should respond to this.

Reply:
I manually checked this before making use of `const` and can confirm that it is being used in several places within `core` already, e.g. `src/opnsense/www/js/opnsense_ui.js` and `src/opnsense/www/js/opnsense_bootgrid_plugin.js`. As they are not preprocessed and delivered like that to the browser, I think this should be fine?

Reply:
They have been removed some time ago and in that case it looks like they came back.
The only issue is that IE yields a syntax error which I do not care about as I don't use it. The bright side is that Windows 7 seems to become EoL in January: https://support.microsoft.com/de-at/help/4057281/windows-7-support-will-end-on-january-14-2020

So please wait for the feedback of a core maintainer.

Reply:
use of `const` is fine indeed, we also have some of them in our libs.

Reply:
Thanks for confirming, @AdSchellevis!